website/source/docs/commands/node-status.html.md.erb

```diff
@@ -149,6 +149,25 @@ ed3665f5  8bf94335  example  cache       run             running
 24cfd201  8bf94335  example  cache       run             running
 ```
 
+You will note that in the above examples, the **Allocations** output contains
+columns labeled **Desired Status** and **Client status**.
+
+Desired Status represents the goal of the scheduler on the allocation with
+the following valid statuses:
+
+- *run*: The allocation should run
+- *stop*: The allocation should stop
+- *evict*: The allocation was evicted and should stop
```

Comment:
I would remove evict for now since we never put an allocation in that state at the moment.